htdocs/js/notebook/cell_view.js

```diff
@@ -368,6 +368,16 @@ function create_cell_html_view(language, cell_model) {
             exec: function() {
                 ace_widget_.blur();
             }
+        }, {
+            name: 'executeAll',
+            bindKey: {
+                win: 'Ctrl-u',
+                mac: 'Command-u', 
+                sender: 'editor'
+            },
+            exec: function() {
+                RCloud.UI.run_button.run();
+            }
```

Comment:
Interestingly, this doesn't seem to be necessary - the global handler will still work if this is removed. 

I noticed this because I was concerned that the key would not work in the prompt or assets. But it does.